suite-common/wallet-utils/src/validation.ts

```diff
@@ -29,7 +29,11 @@ const getCoinFromTestnet = (symbol: Account['symbol']) => {
 export const isAddressValid = (address: string, symbol: Account['symbol']) => {
     const networkType = getNetworkType(symbol);
     const updatedSymbol = getCoinFromTestnet(symbol);
-    return addressValidator.validate(address, updatedSymbol.toUpperCase(), networkType); // TODO(vl): validate address
+
+    // TODO(vl): validate address
```

Comment:
Will revert this commit when [this PR](https://github.com/trezor/trezor-address-validator/pull/25) is merged into the address validator repo.